### PR TITLE
updated PPR UI CD pipeline

### PR DESCRIPTION
**Pull request by flutistar**
*Issue #:* /bcgov/entity###

*Description of changes:*
* Updated PPR-UI-CD
* Updated PPR-UI-CI

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by cameron-eyds — COMMENTED** [review]
Looks good! 
Although I think we want to upgrade the ci in the same manner.

File: .github/workflows/ppr-ui-cd.yml [hunk 1]
```diff
@@ -17,19 +17,14 @@ on:
         - test
         - sandbox
         - prod
-      tagname:
-        description: "Specify a previous version (git tag) to deploy"
-        required: false
-        default: ""
```
Comment (cameron-eyds) [inline .github/workflows/ppr-ui-cd.yml]: Do we need to keep this in order to deploy protected branches?
I'm guessing this is just a label and not required?

**Comment by flutistar** [comment]
> Looks good! Although I think we want to upgrade the ci in the same manner.

Okay, I will upgrade the ci yaml as well.

**Review by cameron-eyds — APPROVED** [review]
LGTM sir!

**Review by doug-lovett — APPROVED** [review]

**Review by pwei1018 — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-2053-4w2lzwqx.web.app

**Pull request merged**